<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="7" hostname="ROCKET" name="com.selenium.demo.selenium.demo.practice.PopUp" tests="8" failures="0" timestamp="15 Jun 2020 11:46:52 GMT" time="13.524" errors="1">
  <testcase name="handlingFrame" time="13.524" classname="com.selenium.demo.selenium.demo.practice.PopUp">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.selenium.demo.selenium.demo.practice.PopUp.handlingFrame(PopUp.java:246)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- handlingFrame -->
  <testcase name="textColor" classname="com.selenium.demo.selenium.demo.practice.PopUp">
    <skipped/>
  </testcase> <!-- textColor -->
  <testcase name="newTab" classname="com.selenium.demo.selenium.demo.practice.PopUp">
    <skipped/>
  </testcase> <!-- newTab -->
  <testcase name="JavascriptAlert" classname="com.selenium.demo.selenium.demo.practice.PopUp">
    <skipped/>
  </testcase> <!-- JavascriptAlert -->
  <testcase name="toolTips" classname="com.selenium.demo.selenium.demo.practice.PopUp">
    <skipped/>
  </testcase> <!-- toolTips -->
  <testcase name="handleNewBrowser" classname="com.selenium.demo.selenium.demo.practice.PopUp">
    <skipped/>
  </testcase> <!-- handleNewBrowser -->
  <testcase name="dragDrop" classname="com.selenium.demo.selenium.demo.practice.PopUp">
    <skipped/>
  </testcase> <!-- dragDrop -->
  <testcase name="browserWindow" classname="com.selenium.demo.selenium.demo.practice.PopUp">
    <skipped/>
  </testcase> <!-- browserWindow -->
</testsuite> <!-- com.selenium.demo.selenium.demo.practice.PopUp -->
